databuilder/publisher/neo4j_csv_publisher.py

```diff
@@ -281,16 +262,17 @@ def create_node_merge_statement(self, node_record: dict) -> str:
         :param node_record:
         :return:
         """
-        params = copy.deepcopy(node_record)
-        params['create_prop_body'] = self._create_props_body(node_record, NODE_REQUIRED_KEYS, 'node')
+        template = Template("""
+            MERGE (node:{{ LABEL }} {key: $KEY})
+            ON CREATE SET {{ PROP_BODY }}
+            {% if update %} ON MATCH SET {{ PROP_BODY }} {% endif %}
```

Comment:
Replace `string.Template` with `jinja.Template` in which if condition is supported. Putting condition clause in template is to help readability

Reply:
thanks for the info